src/Dapr.Actors/Runtime/ActorRuntime.cs

```diff
@@ -29,6 +29,12 @@ public class ActorRuntime
 
         private ActorSettings actorSettings;
 
+        internal static JsonSerializerOptions JsonSerializerOptions { get; } = new JsonSerializerOptions()
+        {
+            PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
+            PropertyNameCaseInsensitive = true,
```

Comment:
Do we want to make this case insensitive?

Reply:
If we expect actors to only talk to other actors then case-sensitive is the right default. 

We chose case-insensitive in ASP.NET because there's a whole galaxy of other clients out there (especially in .NET) that do `PascalCase` by default - and the default that a lot of people are used to is case-insensitive. It was a pragmatic decision on our part because diagnosing a deserialization problem due to casing can be really hard for users.